news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,142 @@
+
+## Changes to Performance Points
+
+In recent years, osu!taiko has lagged behind the other three game modes in terms of raw performance point values (around 40% lower). In addition, the survey indicated that the way performance points are calculated needed to be re-evaluated.
+
+Performance points in osu!taiko are currently calculated by summing the "accuracy pp" and "difficulty pp" components of a score — the following changes should serve to increase their values and improve balance not only between difficulty increasing mods, but their reductive counterparts.
+
+Served alongside the SR rework, these changes aim to accurately reflect the new sentiments of the SR system.
+
+### Globally-applied changes
+
+The final step of performance points calculation involves applying a simple multiplier to the calculated value, which consists of both difficulty and accuracy performance points. This allows for simplicity of balancing changes, and works alongside specific multipliers found within their respective separate difficulty values.
+
+- The HD multiplier has been decreased from **1.10x** to **1.075x**, as a further bonus was placed in difficulty PP, rather than globally.
+- An EZ multiplier of **0.975x** has been added, to further nerf a problematic mod, especially when paired with double-time.
+- The NF multiplier has been removed, as osu!taiko's HP system is different, it is not required nor needed, which also helps prevent sandbaggers in lower ranks.
+
+As well as mod multipliers being nerfed, there community survey highlighted the issues with short maps being either underweighted at a full combo, or overweighted at a misscount.
+
+To combat this, misses will now significantly affect difficulty PP on maps with a max combo below 1000, scaling the miss count with by a multiplier of `1000 / total successful hits`, and can significantly impact short-maps with a higher misscount.
+
+![](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-and-performance-points/effective-miss-count.jpg)
+
+### Changes to Difficulty Performance Points Values
+
+The following changes should result in impressive scores on harder beatmaps rewarding a significant increase of performance point values, with top player [syaron105](https://osu.ppy.sh/users/8741695) gaining over 6400 total PP!
+
+Please note that these changes only affect the difficulty part of performance, which refers to the Star Rating value under a mathematically scaled formula.
+
+- A further EZ multiplier of **0.985x** has been added, further supplementing its nerf.
+- A HR multiplier of **1.05x** has been added, this was added as a way to rebalance the changes found within accuracy PP.
+- Difficulty pp now decreases by the square of decreasing accuracy instead of linearly, which applies a huge nerf unforeseen within previous low-accuracy scores.
```

Comment:
bit of a strange turn of phrase

```suggestion
- Difficulty pp now decreases by the square of decreasing accuracy instead of linearly, which applies a huge nerf to previously overweighted low-accuracy scores.
```